pkg/codegen/pcl/utilities.go

```diff
@@ -236,3 +236,13 @@ func GenerateMultiArguments(
 		}
 	}
 }
+
+func SortedKeys[V any](properties map[string]V) []string {
+	propertyNames := make([]string, 0)
```

Comment:
I'd rephrase the names in this to not reference properties. so s/properties/m s/propertyNames/keys etc